src/components/Top.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import { Button } from 'reactstrap';
+import { Link, withRouter, Redirect} from 'react-router-dom';
+
+class Top extends Component {
+  constructor(props){
```

Comment:
https://github.com/duyoji/oni-on-cheese/pull/56/files#r157937972

same as this.